hack/postrelease/test_windows_artifacts.py

```diff
@@ -11,7 +11,7 @@ def test_calico_release_has_windows_zip():
     assert req.status_code == 302
 
 def test_calico_windows_script_uses_expected_install_zip():
-    resp = requests.get('https://projectcalico.docs.tigera.io/archive/%s/scripts/install-calico-windows.ps1' % RELEASE_STREAM)
+    resp = requests.get('https://github.com/projectcalico/calico/releases/download/%s/install-calico-windows.ps1' % RELEASE_STREAM)
```

Comment:
I think we need to use `RELEASE_VERSION` (vX.Y.Z) here, instead of `RELEASE_STREAM` (just vX.Y)